python-attr-lookup/src/plang/PythonType.java

```diff
@@ -41,15 +41,19 @@ public PythonObject getBase() {
 
     @Override
     protected List<PythonObject> buildMRO() {
-        throw new UnsupportedOperationException("not implemented yet");
+        List<PythonObject> mro = new ArrayList<>();
+        mro.add(this);
+        if (base != null)
+            mro.add(base);
+        return mro;
```

Comment:
Again, I shouldn't have used mro here. Also, yours is significantly more robust. Yours is this:

```python
        if(getBase() != null)
            result.addAll(getBase().getMRO());
        if(getType() != null)
            result.addAll(getType().getMRO());
```

Adding the MRO of the base is obviously better than just adding the base, and it will probably break things to not add the MRO of the type.